pkg/routing/selector/utils.go

```diff
@@ -39,3 +49,23 @@ func LimitsReached(limitConfig config.LimitConfig, nodeStats *livekit.NodeStats)
 
 	return false
 }
+
+func SelectSortedNode(nodes []*livekit.Node, sortBy string) (*livekit.Node, error) {
+	if sortBy == "" {
+		return nil, ErrSortByNotSet
+	}
+
+	// Return a node based on what it should be sorted by for priority
+	switch sortBy {
+	case "random":
+		idx := funk.RandomInt(0, len(nodes))
+		return nodes[idx], nil
+	case "sysload":
+		sort.Slice(nodes, func(i, j int) bool {
+		  return GetNodeSysload(nodes[i]) < GetNodeSysload(nodes[j])
+		})
+		return nodes[0], nil
+	default:
```

Comment:
> can we default to random, the previous default?

It gets defaulted in the config if unset here: https://github.com/livekit/livekit-server/pull/599/files#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R240. I could do it here as well, but it seemed better to catch the default in this way so it would throw an error if there was a typo in a config or some unknown value was set. Do you still want me to also set the default here to random?

Reply:
that makes sense, I agree this is better to catch potential typos 👍